package.json

```diff
@@ -30,22 +30,19 @@
   "devDependencies": {
     "auto-changelog": "^1.16.2",
```

Comment:
It's possible we could upgrade this. The [breaking changes](https://github.com/cookpete/auto-changelog/blob/master/CHANGELOG.md#v200) likely don't affect us

Reply:
Yeah, I think we should be fine...